components/upload/__tests__/uploadlist.test.js

```diff
@@ -264,7 +249,6 @@ describe('Upload List', () => {
     await delay(200);
     expect(wrapper.state().fileList[2].thumbUrl).not.toBeFalsy();
   });
-
```

Comment:
Why delete those blanks?

Reply:
After fighting 'LF' and 'CRLF', I forget reset those file. sorry. but it seem like pass lint after all.

Reply:
You can reset these.

Reply:
All right, mate. I add another commit to restore those blanks. Please review it. Thanks.